Dockerfile.build.template

```diff
@@ -0,0 +1,37 @@
+# Build nodejs dependencies
+FROM resin/%%ARCH%%-node:6.2-slim
```

Comment:
@petrosagg you're already using a slim image, I wasn't suggesting any change there.

As for matching the oe-image version, there definitely needs to be a comment to that effect here, otherwise I can easily see someone changing it without realising that it needs to stay in sync.  For that matter what is the process for upgrading the oe-image version, because in that case I'd suggest updating both to pick up the fixes